### DISCUSSION TITLE
Uncaught Component already initialized

### DISCUSSION THREAD

**Discussion by addtobag** [livewire/livewire#6049] [1. Help]
I'm seeing this error message when clicking a wire:navigate link, has anyone come across the same?

<img width="810" alt="image" src="https://github.com/livewire/livewire/assets/77623201/5f7f6848-ec83-4b36-bc5f-1231d88a8bab">

**Comment by PhiloNL** [comment]
Please check if you are not loading Alpine/Livewire twice.

**Reply by addtobag** [reply]
pretty certain i'm not, i've upgraded from v2, and removed @livewireScripts and inject_assets is set to true in the config.

**Reply by addtobag** [reply]
Removed some scripts and found it's caused by simplebar.min.js

**Reply by ev3rlost** [reply]
Funny, I don't have any external scripts and I still get this message on every page visit. (using Laravel 10, Livewire 3.0.0-beta.5 and Tailwind.)

**Reply by romansh** [reply]
Please, check app.js, I had another js related issue with Alpine.js from V2 code there.

**Reply by ev3rlost** [reply]
In my app.js file I only have the default `import './bootstrap';` import. And my bootstrap.js file is the default installation one.

**Reply by addtobag** [reply]
Are you upgrading from v2? Asking the obvious here but have you removed @livewireScripts and allowed v3 to inject them in the config?

**Reply by ev3rlost** [reply]
I'm working on a fresh V3 installation, I only update the betas to be up-to-date with the bug fixes, so there are no @livewireScripts imports whatsoever from the get-go. The config file was published and the `'inject_assets'` is set to true. It's not that my app won't run as expected, it just annoys me seeing the error in the console - and I'm guessing there should be no errors in the console :)

**Reply by lanstar2** [reply]
I also encountered problems caused by simplebar. Have you solved it?

**Comment by thotam** [comment]
i have same problem too, so i need rollback to v2

**Comment by lanstar2** [comment]
Simplebar covers the DOM, causing the live wire to generate an error reported by the DOM again. @calebporzio

**Comment by rawaby88** [comment]
remove 
`import { Livewire, Alpine } from '../../vendor/livewire/livewire/dist/livewire.esm';
Alpine.plugin(yourCustomPlugin);
Livewire.start();` from app.js

and use old way @livewireStyles @livewireScripts 
and remember to set 

`    'inject_assets' => false,`

**Comment by czelek** [comment]
I have the same error. Got this after I use @added="$refresh" to refresh a component with a list (after I add/delete/update an element of the list). Do you have any solution @addtobag ?
